synoptic/src/synoptic/model/export/JsonExporter.java

```diff
@@ -96,151 +110,326 @@ public EventInstance(int traceID, int eventIndexWithinTrace) {
         }
     }
 
+    // Map of nodes and their respective global ID's
+    static Map<EvBasedNode, Integer> nodesIDMap = new LinkedHashMap<EvBasedNode, Integer>();
```

Comment:
Forgot to mention this in prior emails. Was I supposed to make these private static rather than removing the static ("instead')? I tried doing so and failed, I believe because the methods are static and after encountering errors I read that static methods can't access non-static variables, so I made them all private static. If this is wrong, please let me know and I can fix it.